### ISSUE TITLE
Various minor suggestions

### ISSUE THREAD

**Issue by HDoujinDownloader** [HDoujinDownloader/HDoujinDownloader#105]
Listed below are all suggestions that I haven't created dedicated issues for.

* Colored progress bars (grey for queued, blue for in-progress, green for completed)
* Cover preview visible in the download queue/on hover

**Comment by rs3mk** [comment]
Hi.

Add the option to search on the download page.

Regards

**Comment by ffuentesm** [comment]
Is there a way to select the download location? 
It automatically creates a Manga folder in the download folder, but would be possible to set its own download location?

**Comment by HDoujinDownloader** [comment]
> Is there a way to select the download location? It automatically creates a Manga folder in the download folder, but would be possible to set its own download location?

Yes: On the _**Settings > Save to > General**_ tab, there's a "Default Download Directory" setting that you can change.

**Comment by sambinoTheFlex** [comment]
Hello, 
Is there a way to download entire collections, instead of individual comics? 
Thank you!

**Comment by HDoujinDownloader** [comment]
> Hello, Is there a way to download entire collections, instead of individual comics? Thank you!

It depends on the website. For some websites, you can put tag URLs directly into HDoujin Downloader to add all galleries on that page to the download queue.

If you show me an example of what you'd like to add, I'll see what I can do!

**Comment by ElryGH** [comment]
Remove the download limit, this is not open source if you limit the user to pay to use the total software, its a breach of Githubs License

**Comment by namekman** [comment]
Hi,
It would be nice to have an option to disable the download scheduler once it has been executed.

The reason for that is that, you may just want to shut down the computer once the current downloads are finished,
but the following day, if you forget about the setting and just download one gallery, you may get an undesirable shutdown. (actually happens to me a lot)

Thanks!

**Comment by HDoujinDownloader** [comment]
@namekman Solid reasoning! I'll add a new checkbox in the next update to disable the scheduler on completion.
